projects/bms_carrier/src/fan_control.c

```diff
@@ -0,0 +1,50 @@
+#include "bms.h"
```

Comment:
header convention is:
1. module header (i.e. `"fan_control.h"`)
2. blank line
3. system headers in angle brackets (i.e. `<string.h> or <math.h>`)
4. blank line
5. any other library headers like `soft_timer.h` or whatever else you might need

Reply:
I know `fan_control.h` is redundant but eh convention is convention